README.md

```diff
@@ -47,3 +47,136 @@ Two users will be created by default which are listed below -
     Host User - username: host, password: password
     Participant User - username: participant, password: password
     ```
+
+______________________________________________________________________________________________________________________
+
+# Some Important Commands and their Usage
+
+
+## Challenge
+
+Display all of the challenges                                 
+    ```$
+       evalai challenges 
+    ```
+
+Display the ongoing challenges                                
+    ```$
+        evalai challenges ongoing
+    ```
+    
+Display the future challenges                                 
+    ```$
+        evalai challenges upcoming
+    ```    
+
+Display the past challenges                                
+    ```$
+        evalai challenges past
+    ```
+
+Display details of a challenge with the given ID (1)                                 
+    ```$
+        evalai challenge 1  
+    ```
+    
+Display all challenges that you've participated in                                
+    ```$
+        evalai challenges --participant
+    ```
+
+Display all challenges that you've hosted                                  
+    ```$
+        evalai challenges --host
+    ```    
+
+
+## Challenge Phase
+
+Display all challenge phases of the challenge with Challenge the given ID (1)                                
+    ```$
+        evalai challenge 1 phases 
+    ```    
+
+Display the details of the phase with the given phase ID (4) of the challenge with the given challenge ID (1)                                
+    ```$
+        evalai challenge 1 phase 4
+    ```
+
+
+## Teams    
+
+ Participate in a challenge with the given challenge ID (1) and given team ID (2)                                
```

Comment:
@amanex007 can you make sure lines are consistent. Some lines start with a leading space but some dont. Can you please fix that